src/lib/caching/structures/guilds/GuildMember.ts

```diff
@@ -212,6 +212,22 @@ export class GuildMember extends Structure {
 		return this;
 	}
 
+	/**
+	 * Returns the users avatar url.
+	 * @param options The image size, format and other options.
+	 */
+	public avatarURL(options?: ImageURLOptions): string | null {
+		return this.user?.avatar ? this.client.api.cdn.userAvatar(this.id, this.user.avatar, options) : null;
```

Comment:
Both this and display should return the user one, via `return this.user?.<methodName>(options) ?? null`